objectRecognitionSource/src/main/java/io/vantiq/extsrc/objectRecognition/ObjectRecognitionCore.java

```diff
@@ -647,6 +670,10 @@ public void close() {
             pollTimer.cancel();
             pollTimer = null;
         }
+        if (pool != null) {
+            pool.shutdownNow();
+            pool = null;
+        }
```

Comment:
(Not about this change but about close()) -- Doesn't close need to sync on whatever's being used to snarf the neuralNet variable?  Otherwise, everyone else syncing on it is not of much value.

Reply:
Will add the synch now.